libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/utils/ToleranceUtilsTest.java

```diff
@@ -51,7 +51,7 @@ public void dynamicRerouteDistanceTolerance_userJustPastTheIntersection() throws
     DirectionsRoute route = buildTestDirectionsRoute();
     RouteProgress routeProgress = buildDefaultTestRouteProgress();
     double distanceToIntersection = route.distance();
-    LineString lineString = LineString.fromPolyline(route.geometry(), Constants.PRECISION_6);
+    LineString lineString = LineString.fromPolyline(route.geometry(), PRECISION_6);
```

Comment:
prefer `Constants.PRECISION_6` so that the constant is qualified. That's more legible than a static import imho
just a general hint, goes for all the other files with the PRECISION_6 constant as well of course, but no need to replace all if you don't agree

Reply:
Yeah this is used inconsistently across the whole code base. No need to start fixing this here. Would be good to have a lint for this in the future.